docs/quickstarts/nextjs.mdx

```diff
@@ -46,38 +48,34 @@ pnpm add @clerk/nextjs
 
 </CodeBlockTabs>
 
-### Set environment keys
-
-In your Next.js project's root folder, you may have a `.env.local` file alongside `package.json` and other configuration files. If you don't see it, create it.
+### Set your environment keys
 
-Add the following code to your `.env.local` file to set your public and secret keys.
+1. If you don't have a `.env.local` file in the root directory of your Next.js project, create one now.
+2. Find your secret keys. If you're signed into Clerk, the `.env.local` snippet below will contain your secret key. Otherwise:

```

Comment:
Sorry, I didn't see this before, but this isn't just for their secret key. It's for their publishable key too.